src/JestExt/core.ts

```diff
@@ -144,6 +145,14 @@ export class JestExt {
     };
   }
 
+  private setupIgnoreAction(folder: string): messaging.MessageAction {
```

Comment:
each JestExt/core corresponding to a workspace, you can find it in `this.extContext.workspace.name`. So we don't need to pass this extra argument around from the listeners to here. It should eliminate some API changes for the exit event.

Reply:
oh i was looking into that real quick cause this
```
this.session.context.onRunEvent.fire({
        type: 'exit',
        process,
        workspaceFolder: this.session.context.workspace.name,
      }); 
```
was quite the giveaway they're somwhere close (i didnt look further into it when i didnt find a property called session ^^)

gonna refactor to use this
`this.extContext.workspace.name `